inspirehep/modules/authors/views/holdingpen.py

```diff
@@ -314,16 +314,23 @@ def newreview():
     objectid = request.values.get('objectid', 0, type=int)
     if not objectid:
         abort(400)
-    workflow_metadata.extra_data['is-update'] = True
+
     workflow_metadata = WorkflowUIRecord.get_record(objectid)['metadata']
 
+    # Initialise extra_data
```

Comment:
(I mean that I have this fix also on my branch, just swapping the lines, without having to initialize it)